apps/web/composables/useProductAttributes/__tests__/useProductAttributes.spec.ts

```diff
@@ -0,0 +1,36 @@
+describe('useProductAttributes', () => {
+    it('initializes state correctly', () => {
+        const { attributes, attributeValues, combinations, itemId, variationId } = useProductAttributes();
+
+        expect(attributes.value).toEqual([]);
+        expect(attributeValues.value).toEqual({});
+        expect(combinations.value).toEqual([]);
+        expect(itemId.value).toBe(0);
+        expect(variationId.value).toBe(0);
+    });
+
+    it('updateValue updates attribute values', () => {
+        const { updateValue, attributeValues } = useProductAttributes();
+        updateValue(1, 1);
+
+        expect(attributeValues.value).toEqual({});
+
+        updateValue(1, 2);
+
+        expect(attributeValues.value).toEqual({});
```

Comment:
**todo**: Split into multiple test cases. Every test case should assert against a single outcome, not multiple.

**question**: Why do you always assert against an empty object? I don't understand what we're validating here if the expected value never changes.